config/maintainers.json

```diff
@@ -130,6 +130,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": null
+    },
+    {
+      "github_username": "andres-zartab",
+      "alumnus": true,
+      "show_on_website": true,
+      "name": "Andrés Zarta",
+      "link_text": null,
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "Data Scientist with Operations Research background. Started learning Go to improve my software development skills and ended up loving it for its simplicity and amazing community."
     }
   ]
-}
+}
```

Comment:
Can you undo this change?

Reply:
```suggestion
}

```

Reply:
Note: files committed to git optimally have an empty line at the end

Reply:
Will fix it now!

Reply:
Is it good enough to just accept your commit suggestion?

Reply:
yes you can do that